data/tooling-data.yaml

```diff
@@ -1497,6 +1497,23 @@
   bowtie:
     identifier: 'cpp-jsontoolkit'
 
+- name: cypress-ajv-schema-validator
```

Comment:
changes in the tooling-data file belongs to other issues. Please leave this file intact.